### DISCUSSION TITLE
Issue when viewing Recipies 9.1.518 - Fabric

### DISCUSSION THREAD

**Discussion by shashimalrandika** [shedaniel/RoughlyEnoughItems#1028] [Q & A]
When I view some items it's getting this.

![REI](https://user-images.githubusercontent.com/11777144/181263596-a8834fb3-5bf8-4478-bb21-20f74fb4b078.png)

**Comment by shedaniel** [comment]
REI has to be on the server for the Tag category to work, otherwise you should just ignore it
